infra/common/src/main/java/org/apache/shardingsphere/infra/exception/kernel/syntax/UniqueCommonTableExpressionException.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.infra.exception.kernel.syntax;
+
+import org.apache.shardingsphere.infra.exception.core.external.sql.sqlstate.XOpenSQLState;
+import org.apache.shardingsphere.infra.exception.core.external.sql.type.kernel.category.SyntaxSQLException;
+
+public final class UniqueCommonTableExpressionException extends SyntaxSQLException {
+    
+    private static final long serialVersionUID = -8206891094419297634L;
+    
+    public UniqueCommonTableExpressionException(final String alias) {
+        super(XOpenSQLState.SYNTAX_ERROR, 500, "Not unique table/alias: '%s'", alias);
```

Comment:
Is this exception consistent with the database?

Reply:
If yes, please update the exception document.

Reply:
Yes this Error is present in MySQL  with MySQl error code 1066
and SQLState as 42000 so do I have to create this exception  somewhere else